Gaijinizer patch/Enemies.txt

```diff
@@ -2514,48 +2517,49 @@ eventually it will be an abandoned man.#MTLed
 > BEGIN STRING
 氷に弱い、物理回避が少し高い
 > CONTEXT: Enemies/603/BookAttribute/2
-Weak in ice, physical avoidance is a little high#MTLed
+Vulnerable to Ice, decent at dodging Physical#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ダッキの変身フォームのひとつ。
 > CONTEXT: Enemies/603/BookDescription/4
 > CONTEXT: Enemies/605/BookDescription/4
-One of Dakchi's makeover forms.#MTLed
+One of Daji's many forms.#Ron
 > END STRING
 
 > BEGIN STRING
 魔獣の特性を的確に体現しており、敏捷性、筋力が
 特に強化されている形態となる。
 ちなみに、変身中の知性はバーゲスト並になるようだ。
 > CONTEXT: Enemies/603/BookDescription/5
-It accurately embodies the characteristics of monsters, has
-agility and strength
-It is a form that is particularly enhanced.
-By the way, the intellect during transformation seems to be
-equal to barghest.#MTLed
+A form that perfectly embodies the characteristics
+of magical beasts, with agility and general
+abilities particularly strengthened. Though her
+intelligence in this form also changes to be on
+par with that of a Barghest.#Ron
 > END STRING
```

Comment:
minor changes for flow:
>_**A form that perfectly embodies the characteristics
of magical beasts. While her agility and strength
have been greatly improved, her intelligence in
this form seems to be on par with a Barghest's.**_